packages/emails/src/templates/TeamInviteEmail.tsx

```diff
@@ -65,7 +65,7 @@ export const TeamInviteEmail = (
           {props.language(
             `email_user_invite_subheading_${props.isOrg ? "org" : props.parentTeamName ? "subteam" : "team"}`,
             {
-              invitedBy: props.from,
+              invitedBy: props.from.toString(),
```

Comment:
haven't tested this yet, but this could work

Reply:
`props.from` is a string so not 100% sure what this is trying to do